transport/owr_transport_agent.c

```diff
@@ -2222,6 +2291,21 @@ static gboolean on_sending_rtcp(GObject *session, GstBuffer *buffer, gboolean ea
     return do_not_suppress;
 }
 
+static void on_feedback_rtcp(GObject *session, guint type, guint fbtype,
+    guint sender_ssrc, guint media_ssrc, GstBuffer *fci, gpointer user_data)
+{
+    GstMapInfo mapinfo = { NULL, 0, NULL, 0, 0, { NULL, }, { NULL, } };
```

Comment:
I expect it will be necessary according to compilers and map/unmap do not explicitly assign anything to mapinfo.